## 99_readme_typo.dpatch by dann frazier <dannf@hp.com>
##
## Typo in README. Closes: #508231
Index: trunk/README
===================================================================
--- trunk.orig/README
+++ trunk/README
@@ -24,7 +24,7 @@ Background
 ==========
 I originally wrote ipmitool while between projects and employeed at Sun
 Microsystems.  Sun had just embarked on a new line of general-purpose x86
-servers that inclued an OEM Intel board with an IPMIv1.5 BMC on board.
+servers that included an OEM Intel board with an IPMIv1.5 BMC on board.
 It started with an idea that remote chassis power control would be a handy
 feature for my systems in the lab and from there it grew into a multi-
 purpose tool that lots of people found useful.  I decided to release it
Index: trunk/doc/ipmitool.1
===================================================================
--- trunk.orig/doc/ipmitool.1
+++ trunk/doc/ipmitool.1
@@ -63,7 +63,7 @@ chassis power control.
 IPMI management of a local system interface requires a compatible IPMI
 kernel driver to be installed and configured.  On Linux this driver is
 called \fIOpenIPMI\fP and it is included in standard distributions.
-On Solaris this driver is called \fIBMC\fP and is inclued in Solaris 10.
+On Solaris this driver is called \fIBMC\fP and is included in Solaris 10.
 Management of a remote station requires the IPMI\-over\-LAN interface to be
 enabled and configured.  Depending on the particular requirements of each
 system it may be possible to enable the LAN interface using ipmitool over
